packages/core/src/components/BubbleList/index.vue

```
@@ -80,20 +73,17 @@ function scrollToBottom() {
         // stopAutoScrollToBottom.value = false;
       });
     }
-  }
-  catch (error) {
+  } catch (error) {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)